### DISCUSSION TITLE
<title>Remove account from VScode</title>

### DISCUSSION THREAD

**Discussion by EdisMR** [community/community#39773] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I want to remove my account from the VS code editor but there is no a way to log out in VS code

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in, ignore that message

**Comment by EdisMR** [comment]
Thanks, i found it on Stack overflow but I couldn't remove this post...
Thanks
